example-deployment/docker/reporting-server.dockerfile

```diff
@@ -1,29 +1,32 @@
-FROM rmf-web/builder
+FROM ubuntu:20.04
+ENV TZ=Asia/Singapore
```

Comment:
I'm not sure if we should hardcode this.

---

Can this image be published?

Reply:
> I'm not sure if we should hardcode this.

I think I can send it on build time. I need to do this because if not, on built time a prompt appears asking to pick a TZ, which is not ideal.

> Can this image be published?
Do you mean to our docker registration in github?

Reply:
> Do you mean to our docker registration in github?

I mean make the image in a way that is "deployment agnostic". For example, the hardcoded region info would not make it suitable to be published. If it is possible, then we can move this into the reporting-server package, if we publish it, then the example deployment could download the image instead of building it. If not then the example deployment will refer to the "official" dockerfile to build the image.

Reply:
Ok, now I'm using python as our base image. I think that will be "deployment agnostic", I'm adding now a step to push to the docker registry the image of the reporting server.